runtests.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © 2009- The Spyder Development Team
+# Licensed under the terms of the MIT License
+#
+
+"""
+"""
+
+# Third party imports
+import qtpy
+import pytest
+
+
+def main():
+    pytest.main(['-x', 'spyderlib',  '-v', '-rw', '--durations=10',
+                 '--cov=spyderlib', '--cov-report=term-missing'])
```

Comment:
These options could be set in setup.cfg so that manually running pytest would use them. Honestly I'm not sure what's best, I'm just thinking aloud here.